src/main/java/frc/robot/Calibrations.java

```diff
@@ -30,4 +30,7 @@ public final class Calibrations {
     public static final double TURNING_MIN_OUTPUT = -1;
     public static final double TURNING_MAX_OUTPUT = 1;
   }
+
+  public static final double INTAKE_INTAKING_POWER = 1.0;
+  public static final double INTAKE_EJECTION_POWER = -1.0;
```

Comment:
Are we almost definitely going to want these to be +/-1? Or should they be placeholders?

Reply:
Reasonable values to start with